HOWTOCONTRIBUTE.rst

```diff
@@ -155,10 +155,13 @@ Preparing Pull Requests
 
    Now running tests is as simple as issuing this command::
 
-    $ coverage run --source climpred -m py.test
+    $ pytest climpred
+
+   Check that your contribution is covered by tests and therefore increases the overall test coverage:
 
+    $ coverage run --source climpred -m py.test
```

Comment:
I think you have to follow this with `coverage report` and `coveralls` which will give you a link to paste into your browser to look at coverage. Might want to check this...

It follows the procedure here: https://github.com/bradyrx/climpred/blob/master/.travis.yml

Reply:
ok. I just thought running `pytest` directly is easier to debug the tests. and `coverage` is anyways via `travis` once on github

Reply:
Yes the `pytest` is good for a user testing. If they want a custom page of what functions are covered, they follow the coveralls steps.